scripts-dev/build_debian_packages

```diff
@@ -21,9 +21,9 @@ DISTS = (
     "debian:buster",
     "debian:bullseye",
     "debian:sid",
-    "ubuntu:bionic",
-    "ubuntu:focal",
```

Comment:
we should probably mention we;re dropping support for focal in the changelog.

Reply:
That was a mistake, fixed 😬